konig-etl/src/main/java/io/konig/etl/aws/PrepareToLoadTargetTable.java

```diff
@@ -31,14 +33,24 @@
 public class PrepareToLoadTargetTable implements Processor {
 
 	public void process(Exchange exchange) throws Exception {
-		String dmlScript = exchange.getIn().getHeader("dmlScript", String.class);
-		exchange.getOut().setBody(fileToString(new File("konig/aws/camel-etl/" + dmlScript + ".sql")));
+		String dmlScript = exchange.getIn().getHeader("dmlScript", String.class);	
+		String currDate=currDate();
+		String dml=fileToString(new File("konig/aws/camel-etl/" + dmlScript + ".sql")).replace("{modified}", currDate);
```

Comment:
I think you need to wrap the `currDate` value in a `TO_TIMESTAMP` function to convert the data type from String to Timestamp.